### DISCUSSION TITLE
feat: Edge support more platform

### DISCUSSION THREAD

**Discussion by kalifun** [j178/leetgo#119] [Ideas]
I can't get cookies properly when I use the Mac of Edge

I tried to read the source code and found that the author only implemented Windows to read cookies。

This PR(https://github.com/j178/kooky/pull/2) is going to support:

- [x] Win
- [x] Mac
- [ ] Linux

**Comment by j178** [comment]
Sorry, I didn't receive any notification about that PR from GitHub, not sure if this is a bug. I will take a look later, thanks 🙏

**Reply by kalifun** [reply]
Maybe I will also support the Linux version, hoping to help this project.
